daemons/attrd/attrd_cib.c

```diff
@@ -73,25 +68,14 @@ attrd_cib_replaced_cb(const char *event, xmlNode *msg)
         return;
     }
 
-    if (pcmk_any_flags_set(change_section,
-                           cib_change_section_nodes
-                           |cib_change_section_status)) {
+    if (cib__element_in_patchset(msg, XML_CIB_TAG_NODES)
+        || cib__transient_attrs_in_patchset(msg)) {
```

Comment:
This is more than a refactor since we're changing what we're looking for -- worth calling out in commit message or even its own commit

Reply:
Definitely. I must've been messing around and forgotten to separate it. One commit should keep all the logic the same while dropping the replace callback.